build.sbt

```diff
@@ -703,14 +703,32 @@ lazy val `oteljava-testkit` = project
   )
   .settings(scalafixSettings)
 
+lazy val `oteljava-context-storage` = project
+  .in(file("oteljava/context-storage"))
+  .dependsOn(`oteljava-common`)
+  .settings(munitDependencies)
+  .settings(
+    name := "otel4s-oteljava-context-storage",
+    libraryDependencies ++= Seq(
+      "org.typelevel" %%% "cats-effect-testkit" % CatsEffectVersion % Test,
+    ),
+    Test / javaOptions ++= Seq(
+      "-Dotel.java.global-autoconfigure.enabled=true",
+      "-Dcats.effect.ioLocalPropagation=true",
+    ),
+    Test / fork := true,
+  )
+  .settings(scalafixSettings)
+
 lazy val oteljava = project
   .in(file("oteljava/all"))
   .dependsOn(
     core.jvm,
     `oteljava-metrics` % "compile->compile;test->test",
     `oteljava-metrics-testkit` % Test,
     `oteljava-trace` % "compile->compile;test->test",
-    `oteljava-trace-testkit` % Test
+    `oteljava-trace-testkit` % Test,
+    `oteljava-context-storage`
```

Comment:
Should it be bundled into the main module?

Reply:
I'm leaning towards not. bundling it in the main module forces it to be used, and does not leave a way to disable it afaict